src/app/compare/Compare.tsx

```diff
@@ -531,7 +532,7 @@ function getAllStats(
   for (const stat of stats) {
     for (const item of comparisonItems) {
       const itemStat = stat.getStat(item);
-      const adjustedStatValue = adjustedStats?.[item.id]?.[stat.id];
+      const adjustedStatValue: number | undefined = adjustedStats?.[item.id]?.[stat.id];
```

Comment:
`adjustedStats?: DimAdjustedStats` (on L459)
and typing `DimAdjustedStats` to account for the malleable nature of stat IDs
```ts
DimAdjustedItemStat {
  [statHash: number]: number;
  [statId: string]: number;
```

would allow adjustedStatValue to appropriately pick up the right type without explicitly declaring it

Reply:
Great call, that works.